### ISSUE TITLE
Finetune for image-text retrieval task on COCO

### ISSUE THREAD

**Issue by yxoh** [salesforce/ALBEF#78]
Are the finetune settings exactly the same as in configs/Retrieval_coco.yaml and configs/Retrieval_flickr.yaml?

R@k became lower and lower when I set finetune with this.

**Comment by LiJunnan1992** [comment]
Yes, you should be able to reproduce the paper's results by using the settings in the yaml files.
